server/content-security-policy.js

```diff
@@ -102,44 +102,35 @@ const getContentSecurityPolicyConfig = () => {
         scriptSrc: [UNSAFE_INLINE, UNSAFE_EVAL], // For NextJS scripts
       }),
     };
-  } else if (env === 'production') {
-    if (process.env.WEBSITE_URL === 'https://staging.opencollective.com') {
-      return {
-        reportOnly: false,
-        directives: generateDirectives({
-          imgSrc: [
-            'opencollective-staging.s3.us-west-1.amazonaws.com',
-            'opencollective-staging.s3-us-west-1.amazonaws.com',
-          ],
-        }),
-        reportUri: [
-          'https://o105108.ingest.sentry.io/api/1736806/security/?sentry_key=2ab0f7da3f56423d940f36370df8d625',
+  } else if (env === 'staging') {
+    return {
+      reportOnly: false,
+      directives: generateDirectives({
+        imgSrc: [
+          'opencollective-staging.s3.us-west-1.amazonaws.com',
+          'opencollective-staging.s3-us-west-1.amazonaws.com',
         ],
-      };
-    } else if (process.env.WEBSITE_URL === 'https://opencollective.com') {
-      return {
-        reportOnly: true,
-        directives: generateDirectives({
-          imgSrc: [
-            'opencollective-production.s3.us-west-1.amazonaws.com',
-            'opencollective-production.s3-us-west-1.amazonaws.com',
-          ],
-        }),
-        reportUri: [
-          'https://o105108.ingest.sentry.io/api/1736806/security/?sentry_key=2ab0f7da3f56423d940f36370df8d625',
+      }),
+      reportUri: ['https://o105108.ingest.sentry.io/api/1736806/security/?sentry_key=2ab0f7da3f56423d940f36370df8d625'],
+    };
+  } else if (env === 'production') {
+    return {
+      reportOnly: true,
+      directives: generateDirectives({
+        imgSrc: [
+          'opencollective-production.s3.us-west-1.amazonaws.com',
+          'opencollective-production.s3-us-west-1.amazonaws.com',
         ],
-      };
-    } else {
-      // Third party deploy, or Zeit deploy preview
-      return {
-        reportOnly: true,
-        directives: generateDirectives(),
-      };
-    }
+      }),
+      reportUri: ['https://o105108.ingest.sentry.io/api/1736806/security/?sentry_key=2ab0f7da3f56423d940f36370df8d625'],
+    };
+  } else {
+    // Third party deploy, or Zeit deploy preview
+    return {
+      reportOnly: true,
+      directives: generateDirectives(),
+    };
   }
-
-  // Disabled in other environments
-  return false;
```

Comment:
@Betree That is removed in my version, not sure if you saw it. Let me know what you think is best for `test`, `e2e` and `ci`? And what should be the default?

Reply:
I missed that, perfect!

Reply:
The default should probably be `development` I guess